packages/ui/src/components/sidebar/index.ts

```diff
@@ -0,0 +1,34 @@
+import { SidebarProvider, useSidebar, UseSidebarSignature } from './sidebar-context'
+import { SidebarItem, SidebarItemProps, SidebarMenuSubItem } from './sidebar-item'
+import { SidebarToggleMenuButton } from './sidebar-toggle-menu-button'
+import {
+  SidebarContent,
+  SidebarFooter,
+  SidebarGroup,
+  SidebarHeader,
+  SidebarInset,
+  SidebarMenuSkeleton,
+  SidebarRail,
+  SidebarRoot,
+  SidebarSeparator,
+  SidebarTrigger
+} from './sidebar-units'
+
+const Sidebar = {
+  Root: SidebarRoot,
+  Content: SidebarContent,
+  Footer: SidebarFooter,
+  Group: SidebarGroup,
+  Header: SidebarHeader,
+  Inset: SidebarInset,
+  Item: SidebarItem,
+  MenuSkeleton: SidebarMenuSkeleton,
+  MenuSubItem: SidebarMenuSubItem,
+  Provider: SidebarProvider,
+  Rail: SidebarRail,
+  Separator: SidebarSeparator,
+  Trigger: SidebarTrigger,
+  ToggleMenuButton: SidebarToggleMenuButton
```

Comment:
Just checking, can we add `SidebarSearch` also here?

Reply:
`SidebarSearch` is a part of the app-sidebar component group, which is built above the Sidebar component group. Sidebar component group is a group of primitives, and `SidebarSearch` is a much smarter component; it comes with its own context and does some side effects.

<img width="327" height="251" alt="image" src="https://github.com/user-attachments/assets/9a4a7b75-7c95-496d-9b41-3f1f3a6ea911" />